services/database/vector/capabilities.yaml

```diff
@@ -0,0 +1,91 @@
+shared-capabilities:
```

Comment:
Needs to be updated to the latest schema, same for the shared threats and controls